client/my-sites/sharing/style.scss

```diff
@@ -941,26 +946,26 @@ $color-print: #f8f8f8;
 	line-height: 1;
 }
 
-.sharing-buttons-preview-button__glyph {
+.sharing-buttons-preview-button .social-logo {
 	vertical-align: top;
 	position: relative;
 	top: 3px;
```

Comment:
`3px` top here seems to cause the icon to appear a bit lower than it does on my live site. `2px` looks "best" in my testing. I'd guess it varies browser-to-browser, but on the front-end, the icon appears a bit above center in Chrome and Safari, but about right in Firefox.